lib/tools_qgis.py

```diff
@@ -100,54 +101,28 @@ def qgis_get_layer_source(layer):
     """ Get database connection paramaters of @layer """
 
     # Initialize variables
-    layer_source = {'db': None, 'schema': None, 'table': None, 'service': None,
+    layer_source = {'db': None, 'schema': None, 'table': None,
```

Comment:
Key 'service' cannot be removed because is used in other file. Can you please add it again?

Reply:
ok, i've corrected this in commit: 9947041c0ae0a84...